test_runner/src/main/kotlin/ftl/run/platform/android/CreateInstrumentationConfig.kt

```diff
@@ -15,5 +15,6 @@ internal fun AndroidArgs.createInstrumentationConfig(
     numUniformShards = numUniformShards,
     testShards = testApk.shards.testCases,
     keepTestTargetsEmpty = disableSharding && testTargets.isEmpty(),
-    environmentVariables = testApk.environmentVariables
+    environmentVariables = testApk.environmentVariables,
+    testFlagForShard = testTargetsForShard
```

Comment:
Changing the name from testTargetsForShard into testFlagForShard can be confusing.

Reply:
agreed. It appears i made a mistake here with the name.